### PR TITLE
[TD-12534](connector)<enchance>:scharp connector publish to nuget

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: src/connector/C#/src/TDengineDriver/TDengineDriver.csproj [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
-    <TargetFramework>net5.0</TargetFramework>
+ <PropertyGroup>
+    <TargetFrameworks>net5;netstandard2.0;net45</TargetFrameworks>
+    <PackageId>TDengine.Connector</PackageId>
+    <PackageIcon>logo.jpg</PackageIcon>
+    <Version>1.0.3</Version>
+    <Authors>taosdata</Authors>
+    <Company>www.taosdata.com</Company>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageTags>Taos;Data;Microsoft.NET.Sdk;Iot;bigdata;TDengine;taosdata</PackageTags>
```
Comment (sangshuduo) [inline src/connector/C#/src/TDengineDriver/TDengineDriver.csproj]: IoT or IOT

File: src/connector/C#/src/TDengineDriver/TDengineDriver.csproj [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+    <Description>
+        This is the C# connector's classlib that lets you connect to TDengine. 
+        This C # connector supports: Linux 64/Windows x64/Windows x86.
+        more information please vist:https://www.taosdata.com
```
Comment (sangshuduo) [inline src/connector/C#/src/TDengineDriver/TDengineDriver.csproj]: visit

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments

File: src/connector/C#/src/TDengineDriver/TDengineDriver.csproj (hunk 2 above)
Comment (zitsen) [inline src/connector/C#/src/TDengineDriver/TDengineDriver.csproj]: `:https` to `: https`, add space between url and others

File: tests/examples/C#/stmt/StmtDemo.cs [hunk 3]
```diff
@@ -86,8 +86,8 @@ static void Main(string[] args)
             stmtDemo.ExecuteQuery(createTable);
 
             stmtDemo.StmtInit();
-            string[] tableList = { "stmtdemo" };
-            stmtDemo.loadTableInfo(tableList);
+            // string[] tableList = { "stmtdemo" };
+            // stmtDemo.loadTableInfo(tableList);
```
Comment (zitsen) [inline tests/examples/C#/stmt/StmtDemo.cs:90]: Does this cause error if uncommented?

**Review by zitsen — CHANGES_REQUESTED** [review]
Please update.

File: tests/examples/C#/stmt/StmtDemo.cs (hunk 3 above)
Comment (xleili) [reply tests/examples/C#/stmt/StmtDemo.cs:90]: 执行查询前调用loadTableInfo，会导致select出来的结果为空。所以我注释掉了

File: src/connector/C#/src/TDengineDriver/TDengineDriver.csproj (hunk 2 above)
Comment (xleili) [reply src/connector/C#/src/TDengineDriver/TDengineDriver.csproj]: ok

File: src/connector/C#/src/TDengineDriver/TDengineDriver.csproj (hunk 1 above)
Comment (xleili) [reply src/connector/C#/src/TDengineDriver/TDengineDriver.csproj]: ok

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by zitsen — APPROVED** [review]
LGTM

**Pull request merged**